addons/gdUnit4/test/spy/UnitPreset.gd

```diff
@@ -1,18 +1,18 @@
-class_name UnitPreset
+class_name ClassWithEnumConstructor
 extends RefCounted
 
-enum  Rarity {
-	COMMON = 10,
+enum  EnumValue {
+	ONE = 10,
 	TWO = 20
 }
 
-var _rarity :Rarity
+var _value :EnumValue
 
 # using an enum in the constructor
-func _init(rarity: Rarity, presets_to_filter_out: PackedStringArray) -> void:
-	_rarity = rarity
+func _init(value: EnumValue) -> void:
```

Comment:
do not remove the second parameter, it is just to have an example with more that on argument